packages/react/src/components/DatePicker/DatePicker.js

```diff
@@ -523,31 +521,34 @@ export default class DatePicker extends Component {
     }
   };
 
+  static contextType = PrefixContext;
+  prefix = this.context;
```

Comment:
```suggestion
```

Reply:
@dakahn I think this won't work as a class instance property and will need to be changed. If the context value changes, then `this.prefix` would not be updated.